app/webhooks/configurations.py

```diff
@@ -0,0 +1,16 @@
+from datetime import datetime
+
+from .core import GenericJsonTransformWithHexStrConfig, GenericJsonWithHexStrPayload
```

Comment:
Since we are not using the hex-string parsing features, you could remove the "WithHexStr" part.